MyXamarinApp.API/Controllers/EmployeeController.cs

```diff
@@ -21,21 +23,79 @@ public EmployeeController(IEmployeeRepository employeeRepository)
         public async Task<ActionResult<List<EmployeeModel>>> GetAllEmployees()
         {
             var employees = await _employeeRepository.GetAllEmployees();
-            return employees;
+            if (employees != null)
```

Comment:
is it even possible to get null ? Also for a list there you can send in a empty collection via 200 OK result instead of 404 Not Found